src/redux/features/rocketView.jsx

```diff
@@ -0,0 +1,53 @@
+import '../../styles/rockets.css';
+import { useSelector, useDispatch } from 'react-redux';
+import { reserve, cancel } from './rocketsSlice';
+
+const RocketsView = () => {
+  const dispatch = useDispatch();
+
+  const rocketstate = useSelector((state) => state.rockets);
+  const rocketsData = rocketstate.Data;
+
+  const handleReservations = (id) => {
+    dispatch(reserve(id));
+  };
+
+  const handlecancelation = (id) => {
+    dispatch(cancel(id));
+  };
+
+  return (
+    rocketsData.map((obj) => (
+      <div key={obj.id} className="rocketInfo">
+        <img src={obj.image} alt={`rocket ${obj.id}s display`} className="rocketImage" />
+        <section className="rocketSecInfo">
+          <span className="rocketTitle">{obj.name}</span>
+          {obj.reserved && (
+          <>
+            <div className="bookMarked">Reserved</div>
+            <p className="rocketParagraphbooked">{obj.description}</p>
+          </>
+          )}
+          {!obj.reserved && (
+          <p className="rocketParagraph">{obj.description}</p>
+          )}
+
+          {!obj.reserved && (
+          <button type="submit" className="bookRocket" onClick={() => handleReservations(obj.id)}>
+            Reserve Rocket
+          </button>
+          )}
+
+          {obj.reserved && (
+          <button type="submit" className="cancelRocket" onClick={() => handlecancelation(obj.id)}>
+            Cancel Reservation
+          </button>
+          )}
+
+        </section>
+      </div>
```

Comment:
- [ ] For the best JavaScript practices and the best readability, this part can be imported from a rocket component. Also, this file shouldn't be in the redux directory. Please move it to the components folder like the MissionList.js and Mission.js files: 
![image](https://github.com/gisachris/react-group-project/assets/91486791/cac03e3b-dd70-419b-adde-5f3d024ad853)